src/index.html

```diff
@@ -1,32 +1,33 @@
+
 <!doctype html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8" />
+    <meta charset="UTF-8">
     <meta
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    />
-    <meta
-      http-equiv="X-UA-Compatible"
-      content="ie=edge"
-    />
-    <title>Document</title>
-    <link
-      rel="stylesheet"
-      href="style.css"
-    />
+    >
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Search bar airbnb</title>
+    <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form class="search__form search__icon--size--l" data-qa="big">
+      <input
+        type="text"
+        placeholder="Try &quot;Los Angeles&quot;"
```

Comment:
Please change the placeholder value to `placeholder="Try “Los Angeles“"`, with `&quot;` is breaking tests.